docs/api/browser-window.md

```diff
@@ -1038,10 +1050,13 @@ console.log(win.getBounds())
 
 Returns [`Rectangle`](structures/rectangle.md) - The `bounds` of the window as `Object`.
 
-#### `win.getBackgroundColor()`
+#### `win.getBackgroundColor([format])`
```

Comment:
I'm a bit against allowing the user to pick the format here, can we keep it how it was and always return hex? The user can convert it if they like.

Reply:
I considered this as well, but felt that if a user is setting the background color in a non-hex format, then it's overwhelmingly likely they would prefer to receive it back in that format as well. They could of course convert it themselves, but given that it's a fairly minimal surface area unlikely to change or need ongoing maintenance, I feel it's a better and more consistent DX choice to allow them to set and get with the same formatting options.

Reply:
I guess I'm not really sure what the use cases of `getBackgroundColor()` are that would require parsing the color at all, as opposed to passing it along to another color-parsing system.

Reply:
I'm a [-0](https://www.apache.org/foundation/voting.html) on this change. It's an easy change here and could be convenient so I see the appeal, but since there are pre-existing non-Electron popular ways to convert color strings, this also feels like a new maintenance requirement without much upside.